code/modules/mob/living/carbon/superior_animal/roach/roach_summon.dm

```diff
@@ -0,0 +1,66 @@
+// A one-use item that summon a specific type of roach and tame it.
+/* Here is the current list of choices :
+	- "kampfer" : Default Roach
+	- "unbekannt" : Bluespace Roach
+	- "elektromagnetisch" : EMP Roach
+	- "fuhrer" : Fuhrer Roach
+	- "gluhend" : Glowing Roach
+	- "jager" : Hunter Roach
+	- "kaiser" : Kaiser Roach
+	- "kraftwerk" : Nanite Roach
+	- "seuche" : Support Roach
+	- "panzer" : Tank Roach
+	- "gestrahlte" : Toxic Roach
+*/
+/obj/item/roach_summon
+	name = "bluespace roach beacon"
+	desc = "The result of a collaboration between Soteria and the Cht'mant hive. This beacon allow to summon a single tamed roach."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "uplink_access"
+	item_state = "uplink_access"
```

Comment:
Were is the entropy?

Reply:
I do not know how to do bluespace entropy stuff.